<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiShapeWindow</class>
 <widget class="QDialog" name="MultiShapeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="sagittalLayout">
       <item>
        <widget class="QSlider" name="sagittalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="coronalLayout">
       <item>
        <widget class="QSlider" name="coronalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="transverseLayout">
       <item>
        <widget class="QSlider" name="transverseSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="hideOverlaysComboBox">
     <property name="text">
      <string>Hide overlays</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="modalityComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
